egs2/bn_openslr53/asr1/local/data.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+# general configuration
+stage=0       # start from 0 if you need to start from data preparation
+stop_stage=1
+# inclusive, was 100
+SECONDS=0
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+. utils/parse_options.sh
+
+log "data preparation started"
+
+#mkdir -p ${sbn}
+
+workspace=$PWD
+data_dir='/ocean/projects/cis210027p/dzeinali/espnet/egs2/sbn/asr1/data'
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    log "sub-stage 0: Download Data to downloads"
+
+    cd ${data_dir}
+    idxs=("1", "2", "3", "4", "5", "6" "7" "8" "9" "a" "b" "c" "d" "e" "f")
+    for i in "${idxs[@]}"; do
+        wget https://us.openslr.org/resources/53/asr_bengali_${i}.zip
+        unzip -o asr_bengali_${i}.zip
+        rm -f asr_bengali_${i}.zip
+    done
+    mv asr_bengali/* .
```

Comment:
I don't think `mv` would be efficient, how about directly downloading the data to specific spaces.

Reply:
Is this `mv` necessary?